src/test/kotlin/domain/TierCalculationResultCalculationTest.kt

```diff
@@ -43,7 +43,7 @@ internal class TierCalculationResultCalculationTest {
         assessmentComplexityFactors = mapOf()
       )
 
-      val tier = calculator.calculateTier(protectScores, changeScores)
+      val tier = calculator.calculateTier(protectScores, changeScores, true)
```

Comment:
Looks like all the tests in this file are for female offenders. At the moment I think this is only indicated by the `true` which has been added to the arguments.

Reply:
Yep I've got a few 'male' tests to add